### PR TITLE
Delete tables

### PR DESCRIPTION
**Pull request by chrwm**
Enables deletion of OEP tables.

OEP tables for deletion are read from local folder "delete/".

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -36,18 +36,21 @@
 OEDATAMODEL_API_URL = "https://modex.rl-institut.de"
 
 
-def load_credentials():
-    global OEP_USER, OEP_TOKEN, DATABUS_USER, DATABUS_API_KEY, DATABUS_GROUP
```
Comment (henhuy) [inline main.py:40]: DATABUS vars must stay here.

File: main.py [hunk 2]
```diff
@@ -229,19 +254,30 @@ def get_input(prompt, default):
...
-    upload_folder = get_input("Input folder", "data/")
 
     # nomenclature check
     upload_folder = pathlib.Path(upload_folder)
     nomenclature_path = pathlib.Path(nomenclature_path)
     check_nomenclature_table(check_table, upload_folder)
 
     # upload
-    load_credentials()
+    if delete == "no":
```
Comment (henhuy) [inline main.py:279]: now OEP credentials are loaded anyways.
I think you can remove this section and load OPE credentials above in any case...

**Review by henhuy — APPROVED** [review]
Approve! Some minor changes found.
Thanks for implementation!

File: main.py (hunk 1 above)
Comment (chrwm) [reply main.py:40]: Oki thx!

**Pull request merged**